layout docs (#7651)
sveltejs/kit: documentation/docs/30-advanced/10-advanced-routing.md

You can also put a `+page` directly inside a `(group)`, for example if `/` should be an `(app)` or a `(marketing)` page.

#### Breaking out of layouts

> The root layout — which, if omitted, defaults to `<slot />` — applies to every page of your app. If you want some pages to _not_ have the same layout hierarchy as the rest, then you can put your entire app inside one or more groups _except_ the routes that should not inherit the common layouts.

Reviewer: ```suggestion
The root layout applies to every page of your app — if omitted, it defaults to `<slot />`. If you want some pages to have a different layout hierarchy than the rest, then you can put your entire app inside one or more groups _except_ the routes that should not inherit the common layouts.
```
